crates/gossip/src/config.rs

```diff
@@ -81,8 +81,32 @@ pub struct K2GossipConfig {
     ///
     /// Default: 300,000 (5m)
     #[cfg_attr(feature = "schema", schemars(default))]
+    // #[deprecated(
+    //     since = "0.1.9",
+    //     note = "This is being replaced by a burst mechanism instead."
+    // )]
     pub min_initiate_interval_ms: u32,
 
+    /// A factor applied to the default gossip initiation rate to permit a burst of gossip rounds
+    /// to be accepted.
+    ///
+    /// This is a rate-limiting mechanism that permits nodes that are joining the network and
+    /// needing to sync, to initiate gossip faster than usual.
+    ///
+    /// Default: 3
+    #[cfg_attr(feature = "schema", schemars(default))]
+    pub initiate_burst_factor: u32,
+
+    /// The number of initiation windows that the burst is calculated over.
+    ///
+    /// For example, if this is set to `5`, the initiate interval is set to `1m` and the burst
+    /// factor is set to `3` then in 3 minutes, then in a given 3-minute window, 15 gossip rounds
+    /// can be accepted.
+    ///
+    /// Default: 5
+    #[cfg_attr(feature = "schema", schemars(default))]
+    pub initiate_burst_window_count: u32,
```

Comment:
With this factor, in a 5 min window 15 gossip rounds are accepted.
If the factor wasn't applied, in a 1 min window 3 gossip rounds would be accepted, over 5 min 15 gossip rounds. That's not exactly the same, but do you think we can do without the factor for simplicity's sake?